internal/data.go

```diff
@@ -124,9 +124,14 @@ func (x *TableData) CloneWithAttributesOnly(newRawData []byte) *TableData {
 		UpdatedAt:   x.UpdatedAt,
 		TotalChunks: x.TotalChunks,
 		RawData:     newRawData,
+		ValueSize:   x.ValueSize,
 	}
 }
 
+func (x *TableData) Size() int32 {
```

Comment:
What is the difference between this and `ActualUserPayloadSize`?

Reply:
Convenience.
There is no other size than raw size.
Later we can add CompressedSize and that's it.

Also, len of RawData field is not ActualUserPayloadSize.
ActualUserPayloadSize is more like RawSize field, it's filled before compression and chunking.

Reply:
Yes, that's where I am thinking do we need two? We just need to return `raw_size` now?